src/routes/auth.js

```diff
@@ -172,6 +175,24 @@ router.post('/password',
 	})
 )
 
+router.patch(
```

Comment:
just realised the issue mentions a `PUT` method. The `PATCH` intentional?

Reply:
Yeah it's intentional @jocrah  Since we are changing only one field. I learnt it's better to use patch unless we are doing heavy updates but I can change it if it's what you want.

Reply:
It's fine @itzgeebee. It makes sense, but as I say,  it's cool to communicate on the issue beforehand so we're all aware.

 Because without any such info, it just looks like something different from the issue was done

Reply:
@jocrah Alright.